apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.js

```diff
@@ -1,33 +1,77 @@
 // @ts-check
-const ERR = require('async-stacktrace');
+const asyncHandler = require('express-async-handler');
 import * as express from 'express';
-import * as path from 'path';
-const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
-const _ = require('lodash');
 import AnsiUp from 'ansi_up';
+import { z } from 'zod';
 
 import * as sqldb from '@prairielearn/postgres';
+import { AssessmentQuestionSchema, IdSchema, TopicSchema } from '../../lib/db-types';
 
 const ansiUp = new AnsiUp();
 const router = express.Router();
 const sql = sqldb.loadSqlEquiv(__filename);
 
-router.get('/', function (req, res, next) {
-  debug('GET /');
-  const params = {
-    assessment_id: res.locals.assessment.id,
-    course_id: res.locals.course.id,
-  };
-  sqldb.query(sql.questions, params, function (err, result) {
-    if (ERR(err, next)) return;
-    res.locals.questions = _.map(result.rows, (row) => {
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const questions = await sqldb.queryRows(
+      sql.questions,
+      {
+        assessment_id: res.locals.assessment.id,
+        course_id: res.locals.course.id,
+      },
+      AssessmentQuestionSchema.extend({
```

Comment:
I'd rather declare this at the top level of the file so we don't recreate this object on each request. You can name this something like `AssessmentQuestionRowSchema`.

Reply:
Declared this at the top and named it `AssessmentQuestionRowSchema` in commit 9990136.